From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Tue, 18 Aug 2026 17:14:38 -0400
Subject: [PATCH] cmake: Don't set CMAKE_INSTALL_RPATH

Let CMAKE_INSTALL_RPATH_USE_LINK_PATH handle it completely.

Signed-off-by: Ryan Volz <ryan.volz@gmail.com>
---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 87647d5..4139a76 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -147,12 +147,7 @@ if(APPLE)
 	set(CMAKE_MACOSX_RPATH ON)
 	set(CMAKE_SKIP_BUILD_RPATH TRUE)
 	set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
-	set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR}")
 	set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-	list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_LIBDIR}" isSystemDir)
-	if("${isSystemDir}" STREQUAL "-1")
-		set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR}")
-	endif()
 endif()
 
 if (NOT CMAKE_BUILD_TYPE)
-- 
2.51.0

